ledger/api-server-damlonx/reference-v2/src/main/scala/com/daml/ledger/api/server/damlonx/reference/v2/ReferenceServer.scala

```diff
@@ -61,13 +62,18 @@ object ReferenceServer extends App {
 
   val participantLoggerFactory = NamedLoggerFactory.forParticipant(participantId)
   val participantF: Future[(AutoCloseable, StandaloneIndexServer#SandboxState)] = for {
-    indexerServer <- StandaloneIndexerServer(readService, config, participantLoggerFactory)
+    indexerServer <- StandaloneIndexerServer(
+      readService,
+      config,
+      participantLoggerFactory,
+      SharedMetricRegistries.getOrCreate(s"indexer-$participantId"))
     indexServer <- StandaloneIndexServer(
       config,
       readService,
       writeService,
       authService,
-      participantLoggerFactory).start()
+      participantLoggerFactory,
+      SharedMetricRegistries.getOrCreate(s"ledger-api-server-$participantId")).start()
```

Comment:
Can I recommend we start adding a trailing `,` and newline to multi-line function calls to make the diffs cleaner?

Reply:
yeah, good idea. I'm not yet consistent with doing so. Thanks for the reminder.

Reply:
I think later versions of Scalafmt can do this for us.

Reply:
Addressed by 1227da2.